app/controllers/api/v1/decks_controller.rb

```diff
@@ -1,26 +1,27 @@
 class Api::V1::DecksController < ApplicationController
+  before_action :set_deck, only: [:show, :update, :destroy]
   
   # GET /api/v1/decks
   def index
     @decks = Deck.all
```

Comment:
@vlezya why are you still operating with instance variables inside of controllers?